TA_dataset/bin/dataset_search_command.py

```diff
@@ -436,6 +443,13 @@ def generate(self):
                         "DataSetFunction=makeRequest, destination={}, startTime={}"
                         .format(ds_url, time.time())
                     )
+                    tenant_related_payload = get_tenant_related_payload(
+                        acct_dict.get(ds_acct)
+                    )
+                    ds_payload.update(tenant_related_payload)
+                    logger.info(
```

Comment:
Do we need additional `info` log or it can be `debug`?

Reply:
done replace with debug